wrapper.html

```diff
@@ -130,7 +130,7 @@
 	{% if page.custom_fields.fb_pic_url %}
 	<meta name="twitter:image" content="{{ page.custom_fields.fb_pic_url }}" />
 	{% endif %}
-
+	<meta name="twitter:card" content="summary_large_image">

```

Comment:
I'm not totally sure what would happen if this format is chosen while an image is not set. I would assume it would default back to a no-image or small-default-image format? But it there might be an issue, perhaps this should be moved into the if statement directly above, where it's checking/setting an image

Reply:
Hey both, I'm not not seeing the large card in the twitter validator on [this page using the bugfixes template](https://act.350.org/letter/sms-dont-confirm-powell-senate_copy/) - and digging in a bit I noticed that in the [custom page fields history](https://act.350.org/admin/core/allowedpagefield/?q=url&o=name&ot=asc&show_hidden=1), fb_pic_url is deprecated

Looking at the AK docs, I can't entirely tell if it's save to remove the reference from our templates though: https://act.350.org/docs/manual/developer/customizing_pages.html#wrapper-template-requirements
> "By default, the share meta tag values is drawn from the page contents, or from custom fields that were defined for this purpose in earlier versions of ActionKit, but you can override them using the admin's point-and-click interface."

I don't _think_ we need to worry about the scenario where there wouldn't be an image set though as there should be a fallback in the theme. What do you think about taking this out of the if statement and I can QA on a page with no image?

Reply:
I did check the fallback images too based on your comment @alandmih, and in fact there isn't one. We can work with that for now as it doesn't break anything in the share flow, but something to look at in future! Noted it here: https://github.com/350org/ak_intl_v3/issues/335#issuecomment-1162383790